### PR TITLE
OSS Get started update (addresses #2124 )

### LINKED ISSUE #2124: Revamp Getting Started in 2.0 OSS and Cloud
**Rename Get started docs to Install OSS/Sign up for Cloud:** 

- [x] https://docs.influxdata.com/influxdb/v2.0/get-started/ >> https://docs.influxdata.com/influxdb/v2.0/install/
- [x] https://docs.influxdata.com/influxdb/cloud/get-started/ >>  https://docs.influxdata.com/influxdb/cloud/sign-up/

 

**Create new Get started doc w step-by-step tutorial on how to write, query, visualize and monitor/alert data (should be close to same doc for both OSS and Cloud).** 

Consider the following:
      - One option to do a task on the top/guided level. Perhaps how to do it all w a notebook.
      - Best way to dive into other options. Let's add new issues as needed--wanted to get this down to track.

### PR DESCRIPTION
**Pull request by noramullen1**
Closes #2124 

With notebooks moving to OSS GA soon, we'll be able to update the GS with a more tutorial-focused notebooks path (similar to https://docs.influxdata.com/influxdb/cloud/get-started/) soon.

### CODE REVIEW

File: content/influxdb/v2.0/sign-up.md [hunk 1]
```diff
@@ -0,0 +1,630 @@
+---
+title: Sign up for InfluxDB
```
[the remark this answers is not in the document]
Comment (kelseiv) [reply content/influxdb/v2.0/sign-up.md:2]: for OSS, let's global update sign-up >> install

File: content/influxdb/v2.0/sign-up.md [hunk 2]
```diff
@@ -0,0 +1,630 @@
...
+menu:
+  influxdb_2_0:
+    name: Sign up
+weight: 2
+influxdb/v2.0/tags: [sign-up, install]
+---
+
+The InfluxDB 2.0 time series platform is purpose-built to collect, store,
+process and visualize metrics and events.
+Get started with InfluxDB OSS v2.0 by downloading InfluxDB, installing the necessary
```
Comment (kelseiv) [inline content/influxdb/v2.0/sign-up.md:13]:
```suggestion
Get started with InfluxDB OSS 2.0 by downloading InfluxDB, installing the necessary
```

File: content/influxdb/v2.0/sign-up.md [hunk 3]
```diff
@@ -0,0 +1,630 @@
...
+You are ready to [write or collect data](/influxdb/v2.0/write-data).
+
+{{% note %}}
+To automate the setup process, use [flags](/influxdb/v2.0/reference/cli/influx/setup/#flags)
+to provide the required information.
+{{% /note %}}
+
+{{% /tab-content %}}
+<!------------------------------- END UI Setup -------------------------------->
+{{< /tabs-wrapper >}}
```
Comment (kelseiv) [inline content/influxdb/v2.0/sign-up.md:630]:
```suggestion
{{< /tabs-wrapper >}}

After you’ve installed InfluxDB, you’re ready to [get started working with your data in InfluxDB](/influxdb/v2.0/get-started/).
```

File: content/influxdb/v2.0/get-started.md [hunk 4]
```diff
@@ -1,657 +1,37 @@
...
-To automate the setup process, use [flags](/influxdb/v2.0/reference/cli/influx/setup/#flags)
-to provide the required information.
-{{% /note %}}
-
-{{% /tab-content %}}
-<!------------------------------- END UI Setup -------------------------------->
-{{< /tabs-wrapper >}}
-
-## Next Steps
+After you've [signed up for InfluxDB OSS](/influxdb/v2.0/sign-up/), you're ready to start writing, querying, and visualizing data.
```
Comment (kelseiv) [inline content/influxdb/v2.0/get-started.md:14]:
```suggestion
After you've [installed InfluxDB OSS](/influxdb/v2.0/install/), you're ready to get started. Explore the following ways to work with your data:

- [Collect and write data](/influxdb/v2.0/get-started/#collect-and-write-data)
- [Query data](#query-data)
- [Process data](#process-data)
- [Visualize data](#visualize-data)
- [Monitor and alert](#monitor-and-alert)
```

**Review by kelseiv — CHANGES_REQUESTED** [review]

**Review by sanderson — CHANGES_REQUESTED** [review]
Some suggestions, but also an overall critique of new get started content: Rather than link to all the different ways a user can collect/write/query/visualize data, we should walk them through a proven path that will allow them to do all of these things. I would suggest the following:

- Use one of the sample data sets. Walk a user through writing the data set (CSV or LP upload). Or we could have them set up a scraper and use that data.
- Walk through how to query that data in the UI.
- Walk through creating a dashboard to visualize the data.
- etc.

At the end of each section or at the end of the doc, we can provide links to other methods to accomplish each task, but a user should be able to go through the get started guide step by step and successfully complete each task.

**Comment by noramullen1** [comment]
> Some suggestions, but also an overall critique of new get started content: Rather than link to all the different ways a user can collect/write/query/visualize data, we should walk them through a proven path that will allow them to do all of these things. I would suggest the following:
> 
> * Use one of the sample data sets. Walk a user through writing the data set (CSV or LP upload). Or we could have them set up a scraper and use that data.
> * Walk through how to query that data in the UI.
> * Walk through creating a dashboard to visualize the data.
> * etc.
> 
> At the end of each section or at the end of the doc, we can provide links to other methods to accomplish each task, but a user should be able to go through the get started guide step by step and successfully complete each task.

Hey @sanderson , I do agree with that but as stated in the PR description, this content will be replaced with the much more comprehensive notebooks path as soon as notebooks are released for OSS, which should be in the next week or two. Because of that, I used the existing getting started content (it's not actually "new", just moved) for now rather than reinventing the wheel for something that will be up for just a couple weeks.

File: content/influxdb/v2.0/install.md [hunk 5]
```diff
@@ -0,0 +1,632 @@
...
+The InfluxDB 2.0 time series platform is purpose-built to collect, store,
+process and visualize metrics and events.
+Get started with InfluxDB OSS 2.0 by downloading InfluxDB, installing the necessary
+executables, and running the initial setup process.
+
+The primary differences between InfluxDB OSS 2.0 and InfluxDB Cloud are:
+
+- [InfluxDB scrapers](/influxdb/v2.0/write-data/no-code/scrape-data/) that collect data from specified
+  targets are not available in {{< cloud-name "short" >}}.
+- {{< cloud-name "short" >}} instances are currently limited to a single organization.
```
[the remark this answers is not in the document]
Comment (noramullen1) [reply content/influxdb/v2.0/install.md:19]: made another issue for this one: https://github.com/influxdata/docs-v2/pull/2642#discussion_r644313767

**Review by kelseiv — APPROVED** [review]

**Comment by sanderson** [comment]
🤦🏻‍♂️ My bad. I should've read the description. Looks tood.

**Review by sanderson — APPROVED** [review]

**Pull request merged**